README.md

```diff
@@ -31,6 +31,13 @@ You are not required to install a binutils package, but it does speed up the ini
 3. Run `make` in the main directory.  
     **a.** Use the `-jN` argument to use `N` number of threads to speed up building. For example, if you have a system with 4 cores / 4 threads, you should do `make -j4`.
 
+### Building on MacOS
+1. `brew install gcc capstone coreutils`
+2. Put GNU-coreutils `bin` on path using by adding `PATH="/usr/local/opt/coreutils/libexec/gnubin:$PATH"` to your shell rcfile (or update any GNU-util usage in `Makefile` and `tools/Makefile` to use the `g`-prefixed versions).
```

Comment:
When I install homebrew, I don't get this `/usr/local/opt/coreutils/libexec` path. You can use `brew --prefix` to get the exact path or narrow down coreutils path to `brew --prefix coreutils`:
```sh
PATH="`brew --prefix coreutils`/libexec/gnubin:$PATH"
```

I also recommend including working on using `gmake` instead of `make`. Default `make` with macOS Xcode CLI is 3.81 which is too old for the Makefiles in this repo.